plone/app/discussion/browser/moderation.pt

```diff
@@ -229,7 +229,7 @@
                         />
                       </td>
                       <td>
-                        <span tal:content="python:item.author_name or item.Creator">Name</span>
```

Comment:
Make sure we don't lose the "or item.Creator" part in the new code. If the comment doesn't have author_name set for some reason, but it does have a Creator, we should show the creator.